client/src/components/SearchBar/SearchBar.jsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
 const SearchBar = function() {
+  const [ searched, setSearched ] = useState('');
 
+  useEffect(() => {
+    if (searched.length >= 3) {
+      axios.get('http://52.26.193.201:3000/qa/5')
+        .then(res => console.log(res.data))
+        .catch(err => console.log(err));
+    }
+  }, [searched]);
+
+  const handleChange = function(event) {
```

Comment:
Are you wanting to search for an item every time the input is changed or would you rather it do one search once the submit is handled? I used handleChange to update my state and handleSubmit to execute the search

Reply:
Oh. I meant to take out the useEffect for now because it's something I'm going to figure out later along with integration testing between components. Ultimately, useEffect will be needed to perform a get request to the API to update the questions rendered by QuestionList once at least 3 characters are typed in the search bar.